### PR TITLE
Feature/519 individual non contribution

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
[App#519](https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/fecgov/fecfile-web-app/519)

### CODE REVIEW

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+                "FIELD_DESCRIPTION": "AGGREGATION GROUP",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "NON_CONTRIBUTION_ACCOUNT",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_purpose_descrip": {
```
Comment (toddlees) [inline schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:448]: must be "Non-contribution Account Receipt" constant

**Review by toddlees — CHANGES_REQUESTED** [review]

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+            "const": "SA17",
+            "examples": [
+                "SA17"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "SA17",
+                "VALUE_REFERENCE": "SA17",
```
Comment (dheitzer) [inline schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:40]: Should be "SA17 Only" from the spec

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+            "pattern": "^[ -~]{0,2}$",
+            "examples": [
+                "WA"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTOR STATE",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "WA",
+                "VALUE_REFERENCE": null,
```
Comment (dheitzer) [inline schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:349]: Per spec should be "AK,AL,...,ZZ"

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json [hunk 4]
```diff
@@ -0,0 +1,581 @@
...
+            "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$",
+            "examples": [
+                "2018-11-13"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION DATE",
+                "TYPE": "NUM-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 20120615,
+                "VALUE_REFERENCE": null,
```
Comment (dheitzer) [inline schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:388]: Should be "YYYYMMDD"

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json [hunk 5]
```diff
@@ -0,0 +1,581 @@
...
+            "const": "NON_CONTRIBUTION_ACCOUNT",
+            "examples": [
+                "NON_CONTRIBUTION_ACCOUNT"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "AGGREGATION GROUP",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "NON_CONTRIBUTION_ACCOUNT",
+                "VALUE_REFERENCE": null,
```
Comment (dheitzer) [inline schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:443]: Value/rule reference should be "Non-Contribution Account"

**Review by toddlees — APPROVED** [review]

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json (hunk 3 above)
Comment (Elaine-Krauss-TCG) [reply schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:349]: Addressed

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json (hunk 4 above)
Comment (Elaine-Krauss-TCG) [reply schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:388]: Addressed

File: schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json (hunk 5 above)
Comment (Elaine-Krauss-TCG) [reply schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json:443]: Addressed

**Review by mjtravers — APPROVED** [review]

**Review by dheitzer — APPROVED** [review]

**Pull request merged**